bin/runtime/src/lib.rs

```diff
@@ -401,6 +406,8 @@ impl pallet_vesting::Config for Runtime {
     type BlockNumberToBalance = ConvertInto;
     type MinVestedTransfer = MinVestedTransfer;
     type WeightInfo = pallet_vesting::weights::SubstrateWeight<Runtime>;
+    // Maximum number of vesting schedules an account may have at a given moment
+    const MAX_VESTING_SCHEDULES: u32 = 42;
```

Comment:
seem arbitrary :P

Reply:
Again: 

> I think it is copied from Substrate full node, not sure what should be a correct way to decide what value should take such variable

Reply:
they have it equal to `28` at the moment, since we follow polkadot in other places I lowered it to the exact same number and added a suitable comment.